.env

```
@@ -0,0 +1,8 @@
+#######################################
+# End To End Test
+#######################################
+# ⚠️ Required for E2E tests to run
+
+# Cypress Integration Tests
+CYPRESS_INTEGRATION_TEST_PRIVATE_KEY=
+CYPRESS_INTEGRATION_TESTS_INFURA_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)